### DISCUSSION TITLE
YouTube Video Metadata

### DISCUSSION THREAD

**Discussion by Shagnikpaul** [s-frei/TrackSearch#16] [Q&A] [answered]
How to get YouTube video details like channel name, views ? 

By the way great Library ! Keep it up.

**Comment by s-frei** [comment, marked as answer]
Hey there,
at the moment this is not implemented. I focused to get the *important* (track related) information.
I could add some kind of metadata object to the tracks to provide that information.
What information you are interested in ? Only the already described ?

I appreciate that you like it!

**Reply by Shagnikpaul** [reply]
Yup Channel Name , views count , subscribers and anything else you like.

**Comment by s-frei** [comment]
Well, I know added following:
- Channel name
- Channel URL
- Stream Amount (views)
- Thumbnail URL

Available here: [Release 0.6.3](https://github.com/s-frei/TrackSearch/releases/tag/0.6.3) and on MavenCentral

I haven't added information like *subscribers* since it would lead to another request and the aim of this
library is about resolving the *audio stream* and *related metadata* :yum:
I need to keep it simple and maintainable ... paths to information sometimes change.

I hope you are fine with that.

**Reply by Shagnikpaul** [reply]
Thanks a lot for updating ! Those metadatas  will be enough for me.
